source/standards/publish-opensource-code.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: How to publish open source code
+expires: 2018-09-01
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+At GDS we follow the Service Manual, [making source code open and reusable][].
+
+Our open source projects were developed for use on GOV.UK and other work we do, such as [Puppet module for go-audit][] or the [GDS Operations open source web site][]. Unless stated otherwise, all our open source software is released under the [MIT licence][] and is free to use. We encourage and appreciate contributions.
```

Comment:
Perhaps we can make a distinction here between "coding in the open" and open source. 

The things described in the gds-operations guidelines were written for open source libraries. Some of the points aren't relevant to open code (like the changelog, link to a public list of known issues and bugs, security email address, semantic versioning).

Perhaps https://gds.blog.gov.uk/2012/10/12/coding-in-the-open is good to refer to, it explains the difference too.

Reply:
Thanks @tijmenb I'll reword some of the opening paragraphs to make that distinction.